aeron/logging/logging.go

```diff
@@ -86,20 +101,23 @@ func newAeronEncoderConfig() zapcore.EncoderConfig {
 	}
 }
 
-// Default config
-func aeronLoggingConfig() zap.Config {
-	return zap.Config{
-		Level:       zap.NewAtomicLevelAt(zap.InfoLevel),
-		Development: false,
-		Sampling: &zap.SamplingConfig{
-			Initial:    100,
-			Thereafter: 100,
-		},
-		Encoding:         "console",
-		EncoderConfig:    newAeronEncoderConfig(),
-		OutputPaths:      []string{"stderr"},
-		ErrorOutputPaths: []string{"stderr"},
-	}
+func SetConfigAndRebuildAllLoggers(c zap.Config) {
+	defaultConfig = c
```

Comment:
I suppose this is not thread safe, but intention was to use this on application bootstrap only once before an application has really been build and is running. If that is a problem, happy to fix.